client/app/_components/ui/field/DateField.tsx

```diff
@@ -0,0 +1,84 @@
+import { useState } from 'react'
+import { format } from 'date-fns'
+import { Control, Controller, FieldValues, Path } from 'react-hook-form'
+import { BiCalendarAlt } from 'react-icons/bi'
+import {
+  Popover,
+  PopoverTrigger,
+  PopoverContent,
+  FormControl,
+  FormField,
+  FormItem,
+  FormLabel,
+  FormMessage,
+  CustomDayPicker,
+} from '@/_components/ui'
+
+type DateFieldProps<TFormValues extends FieldValues> = {
+  control: Control<TFormValues>
+  name: Path<TFormValues>
+}
+
+const DateField = <TFormValues extends FieldValues>({
```

Comment:
Please add a storybook!

Reply:
Fixed it
https://github.com/HAK2024/fairy-share/pull/80/commits/10a022a763567e66fc5f340c7d30db3a50be4af6